.github/workflows/release.yml

```diff
@@ -9,8 +9,28 @@ env:
   GRADLE_OPTS: "-Dorg.gradle.parallel=true -Dorg.gradle.caching=true"
 
 jobs:
+  preprocess_release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Pre-process Release Name
+        id: pre_process_release_name
```

Comment:
nit: preprocess vs pre_process

...only because it feels ripe for a future bug in this case!